pkgs/by-name/hx/hxn/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  stdenv,
+  darwin,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hxn";
+  version = "1.0.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-zAqYaPtbXqC1YFzCL8EwE1HhuSqVl5lAfnAftwBvnoE=";
+  };
+
+  cargoHash = "sha256-wza/N0+hux5OHnvsLAPwTQLmJqYEr1EjgjpF6k32Yb8=";
+
+  buildInputs = lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.Security ];
+
+  meta = with lib; {
+    description = "Blazingly fast tool to grab screenshots of url/webpages from terminal.";
+    homepage = "https://crates.io/crates/hxn";
```

This feels like a more appropriate homepage
```suggestion
    homepage = "https://github.com/pwnwriter/haylxon";
```

---

yes